src/spectral.jl

```diff
@@ -264,3 +264,25 @@ function contract(nbt::Nonbacktracking, edgespace::Vector)
     contract!(y,nbt,edgespace)
     return y
 end
+
+"""spectral_distance(G₁, G₂ [, k])
+Compute the spectral distance between undirected graphs
+G₁ and G₂ using the top k greatest eigenvalues. If k is
+ommitted, uses k = min(|G₁|,|G₂|).
+
+For further details, please refer to:
+
+JOVANOVIC, I.; STANIC, Z., 2014. Spectral Distances of
+Graphs Based on their Different Matrix Representations
+"""
+function spectral_distance(G₁::Graph, G₂::Graph, k::Integer)
+  A₁ = adjacency_matrix(G₁)
+  A₂ = adjacency_matrix(G₂)
+
+  λ₁ = k < nv(G₁)-1 ? eigs(A₁, nev=k, which=:LR)[1] : sort(eigvals(full(A₁)), rev=true)[1:k]
+  λ₂ = k < nv(G₂)-1 ? eigs(A₂, nev=k, which=:LR)[1] : sort(eigvals(full(A₂)), rev=true)[1:k]
```

Comment:
Maybe refresh the page? I was seeing the outdated diff too on github, for
some reason they are not in sync sometimes.

-Júlio